utils/daos_build.py

```diff
@@ -1,26 +1,52 @@
 """Common DAOS build functions"""
-from SCons.Script import Literal
+from SCons.Subst import Literal
```

Comment:
(pylint-no-name-in-module) No name 'Subst' in module 'SCons'
(pylint-import-error) Unable to import 'SCons.Subst'
(pylint-no-name-in-module) No name 'Subst' in module 'SCons'
(pylint-import-error) Unable to import 'SCons.Subst'